src/Microsoft.AspNet.Mvc.TagHelpers/IHostingEnvironmentExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using Microsoft.Framework.FileSystemGlobbing.Abstractions;
+
+namespace Microsoft.AspNet.Hosting
+{
+    /// <summary>
+    /// Extension methods for <see cref="IHostingEnvironment"/>.
+    /// </summary>
+    public static class IHostingEnvironmentExtensions
```

Comment:
Veryyy high bar for extension methods in MVC, this class can be removed and extension method inlined with the fix to: https://github.com/aspnet/Mvc/pull/1967/files#r24227497